ElectronClient/gui/NoteToolbar/NoteToolbar.tsx

```diff
@@ -49,16 +49,17 @@ function NoteToolbar(props:NoteToolbarProps) {
 		const output = [];
 
 		output.push(
-			cmdService.commandToToolbarButton('historyBackward')
+			cmdService.commandToToolbarButton('historyBackward'),
 		);
 
 		output.push(
-			cmdService.commandToToolbarButton('historyForward')
+			cmdService.commandToToolbarButton('historyForward'),
```

Comment:
Please remove these changes

Reply:
My bad, I will update it tonight.
I thought it was the linter hook that added it but it was probably just my editor 😄

Reply:
Ok, so it was really the husky linter hook that added them and to push without them as you recommend, 
I had to temporarily delete the hook from the package.json.

Totally new on the project so tell me if I should have use another easier solution :smile_cat: 

Running `npm lint` from the root of the repository to test, it updated 65 files.
Did I miss a configuration somewhere?